### ISSUE TITLE
Auth Security Features not working on Linux

### ISSUE THREAD

**Issue by MohammadShabib** [ente-io/ente#1793]
### Description

Ubuntu 20.04
Lockscreen and Email Verification aren't working. 
## Lockscreen
When I click on Lockscreen  a GUI animation appears (enables it and disables it immediate)
after checking logs 

`
[AuthUtil] [INFO] [2024-05-21 09:24:48.267756] Requesting authentication
 [email_util] [SEVERE] [2024-05-21 09:24:52.615552] email sender failed
⤷ type: MissingPluginException
⤷ error: MissingPluginException(No implementation found for method send on channel flutter_email_sender)
⤷ trace: #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:332)
<asynchronous suspension>
#1      _sendLogs (package:ente_auth/utils/email_util.dart:129)
<asynchronous suspension>
#2      sendLogs.<anonymous closure> (package:ente_auth/utils/email_util.dart:58)
<asynchronous suspension>
#3      _ButtonChildWidgetState._onTap.<anonymous closure> (package:ente_auth/ui/components/buttons/button_widget.dart:401)
<asynchronous suspension>
#4      _ButtonChildWidgetState._onTap (package:ente_auth/ui/components/buttons/button_widget.dart:400)
<asynchronous suspension>
`
When I tried multiple times the previous logs stoped showing and only this logs is printed
` [AuthUtil] [INFO] [2024-05-21 09:35:28.941076] Requesting authentication`

## Email Verification
I  have  Email verification enabled but I can't disable it from Linux application when i try to disable it it show same animation as above and from logs  it just shows this
` [AuthUtil] [INFO] [2024-05-21 09:30:10.799546] Requesting authentication`

When I disable Email verification from phone application and reopen the Linux app it still shows its enabled but when i click to disable it it works but I can't reasonable it again
so in summary Email verification doesn't work too, (it only fetches the currently state of this flag - you have to do it manually- )

### Version

3.0.3

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Linux

**Comment by danielktaylor** [comment]
I'm not sure how to enable a lockscreen on the desktop linux app. When I try I get this:

![image](https://github.com/ente-io/ente/assets/919039/0c938ded-9591-4e28-b53e-2988a5516454)

**Comment by ua741** [comment]
This should be fixed in the latest version.
